charters/rust-core/README.md

```diff
@@ -0,0 +1,277 @@
+
+- Strong interoperability assurances.
+- Reduce maintenance burden for particularly dense feature sets (ex. did:dht).
+- Serve as a Schelling Point for matters of implementation governance.
+- Contain a convenient location for a historical record of decisions-made.
+
+Rust is uniquely qualified as the solution for a cross-platform core implementation because of open source tooling availability through WASM and UniFFI. Furthermore, Rust has a broad array of open source support, and Rust has the unique feature of memory safety (relative to alternatives such as C).
+
+## Scope
+
+The comprehensive implementations of the Web5 and tbDEX specifications across Rust, Kotlin and Swift are within the scope of this Working Group. Alongside of each implementation, documentation comments ("doc comments"), test vectors, example usages, and CI/CD solutions (eg. security analysis, test automation, artifact version & publication, etc.) are all within scope of this Working Group. 
+
+The Rust Core Working Group will make an intentional effort to prioritize matters of security and performance. Furthermore, with regards to API design & developer experience (DX), this Working Group will prioritize first and foremost modularity and extensibility, over DX convenience, in order to limit the liklihood of introducing breaking changes to consumers of the delivered SDKs.
+
+The Rust Core Working Group is not concerned with the definitions of the Web5 and tbDEX specifications, both of which are the concerns of the respective Working Group. However, the Rust Core Working Group will offer valuable concrete feedback to the Web5 and tbDEX Working Groups. Although the Web5 and tbDEX specifications are not within the scope of this Working Group, a considerable amount of cross-Working Group collaboration is to be expected (see [Coordination](#coordination) and [Communication](#communication) below).
+
+The Rust Core Working Group codifies a standard API Design ([Web5 APID](#deliverable-1-web5-api-design-apid) & [tbDEX APID](#deliverable-6-tbdex-api-design-apid)) which is implemented across all supported language implementations (Rust, Kotlin and Swift), but any additional API features or design characteristics are not within the scope of this Working Group. Each implementation must support the standard API Design as a baseline implementation, but does not exclude the possibility of additional features.
+
+The Rust Core Working Group will codify itself across the following GitHub repos:
+
+- Web5: https://github.com/TBD54566975/Web5-rs/
```

Comment:
is the deliverable list missing web5 / tbdex wasm binding / JS implementation? forgive me, i don't know how wasm works - but you mentioned wasm above as one of the toolings to enable bindings

Reply:
WASM is intentionally excluded for the time being. We have plenty to do with the existing set of deliverables, so we should do what's here in full whereafter we can amend the charter to include WASM if we deem it worth including.